CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -249,7 +249,11 @@ const char *_CFProcessPath(void) {
 
 #if TARGET_OS_MAC || TARGET_OS_WIN32 || TARGET_OS_BSD
 CF_CROSS_PLATFORM_EXPORT Boolean _CFIsMainThread(void) {
+#if !defined(__OpenBSD__)
```

Comment:
It's a bit clearer if we have the OpenBSD case come first and not invert the condition.
```
#if defined(__OpenBSD__) // special case for OpenBSD
    return pthread_equal(pthread_self(), _CFMainPThread) != 0;
#else // everything else
    return pthread_main_np() == 1;
#endif
```
I usually think of `else` indicating anything not special-cased, so having the else only catch the OpenBSD case seems a bit odd.
Thoughts?

Reply:
Reordered the conditional.